### PR TITLE
INTLY-3681 - uninstall OSD pipeline

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                ansibleTower(
+                  towerServer: 'Dev Tower',
+                  jobTemplate: 'Integreatly_Bootstrap_and_Uninstall_[OSD]',
+                  templateType: 'workflow',
+                  importTowerLogs: true,
+                  removeColor: false,
+                  verbose: true,
+                  extraVars: """---
+                    tower_environment: dev
+                    rhsso_namespace: openshift-sso
```
Comment (tremes) [inline jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Not sure about this line...just stole it from the template definition.

**Comment by tremes** [comment]
Successful run is at https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/osd-cluster-integreatly-uninstall/5/. In fact the previous run was successful too, but I had some typo in the pipeline definition.

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml (hunk 1 above)
Comment (trepel) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Me neither, probably ask PaulMc or Steven.

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+- job:
+    name: osd-cluster-integreatly-uninstall
+    display-name: 'OSD Cluster Integreatly Uninstall'
+    project-type: pipeline
+    concurrent: false
```
Comment (trepel) [inline jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Ok for now, however, we should probably create a followup JIRA ticket to allow for concurrent builds once two issues linked in https://issues.jboss.org/browse/INTLY-3254 are finished. If that makes sense I will create the ticket

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          }
+
+          stage('Delete users') {
+            sh """
+              oc login https://console.${clusterName}.openshift.com --token=${openShiftToken}
+              oc delete user customer-admin
+            """
+
+            sh '''
+              oc delete user $(oc get user | grep tester | awk '{print $1}')
```
Comment (trepel) [inline jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Not sure about `tester` being hard-coded here. I think it should be parametrized with `tester` default value.

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml (hunk 1 above)
Comment (trepel) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Or leave as is, it does not seem that we will be using a different value for rhsso namespace, but still, better to know why it is there.

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml (hunk 3 above)
Comment (tremes) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Ohh yeah right. Thx

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml (hunk 2 above)
Comment (tremes) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: Yes I agree. OK thanks. Feel free to create it.

**Comment by tremes** [comment]
Added the param for username. Successful run https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/osd-cluster-integreatly-uninstall/11

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml (hunk 2 above)
Comment (trepel) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml]: created: https://issues.jboss.org/browse/INTLY-3716

**Comment by trepel** [comment]
LGTM, merging

**Pull request merged**